src/Parachain.sol

```diff
@@ -11,6 +11,11 @@ abstract contract Parachain {
 
     XcmTransactorV2 private constant xcmTransactor = XCM_TRANSACTOR_V2_CONTRACT;
 
+    // The amount of weight an XCM operation takes. This is a safe overestimate
```

Comment:
Possibly provide inline docs link where it shows this is an overestimate. Or do you have a source @tkernell and I could read? Definitely want to make sure these constants won't need to be updated once contracts are deployed

Reply:
> @pawanbisht62 Looking good, just the CI is failing. Also, what docs/resources do y'all have, so we have a way to reproduce these numbers, and so we can ensure the constants you've added won't need to be updated by the parachain owner in the future?

Hi @oraclown, thanks for the review, for the hardcoded values we have just roughly estimated these based on the reads & writes performed by an extrinsic and the cost of I/O operation are considered from this doc (https://docs.moonbeam.network/builders/interoperability/xcm/fees/).

In this [task](https://github.com/tellor-io/tellor-pallet/issues/3), I'm writing benchmarks for each extrinsic which will give us exact weights.

Reply:
The [XCM-Transactor](https://docs.moonbeam.network/builders/interoperability/xcm/xcm-transactor/) pallet is allowing the contracts to call the pallet via xcm using Solidity precompiles: "the XCM-Transactor Pallet allows users to perform remote cross-chain calls through different methods that dispatch the call from different derivated accounts so that they can be easily executed with a simple transaction.". The XCM-Transactor pallet is also available on other EVM smart contract enabled parachains.

The [Xcm section](https://docs.moonbeam.network/builders/interoperability/xcm/) as a whole is probably worth a read for additional context.

Reply:
Thanks for the extra info/resources, y'all.